fe/fe-core/src/main/java/com/starrocks/catalog/Function.java

```diff
@@ -105,6 +106,9 @@ public enum CompareMode {
     @SerializedName(value = "argTypes")
     private Type[] argTypes;
 
+    @SerializedName(value = "argNames")
+    private String[] argNames;
+
```

Comment:
BTW, why don't support a special class to resolve arguments? may be we need support all function in future? separate to handle the arguments and name isn't a sustainable practice

Reply:
and if we need support postiation arguments and name arguments at sametime, define a unified abstraction is better?

Reply:
Adding named argumetns seems change a few codes, seems no need add new class.